### PR TITLE
Separate database svc and minio svc

### PR DESCRIPTION
**Pull request by nowinkeyy**
The database and object-storage deploy the same monitoring service respectively, and the service code needs to be split, and the code optimization has not progressed.

### CODE REVIEW

**Comment by fanux** [comment]
I don’t understand what this PR is used for. You need to add a more detailed explanation.

File: service/database/server/server.go [hunk 1]
```diff
@@ -1,202 +1,38 @@
...
-		http.Error(rw, fmt.Sprintf("Authentication failed (%s)", err), http.StatusInternalServerError)
-		log.Printf("Authentication failed (%s)\n", err)
-		log.Printf("Kubeconfig (%s)\n", pr.Pwd)
+	hs := &http.Server{
+		Addr:    c.Server.ListenAddress,
+		Handler: ps,
 	}
 
-	res, err := ps.Request(pr)
+	listener, err := net.Listen("tcp", c.Server.ListenAddress)
```
Comment (fanux) [inline service/database/server/server.go:27]: Why not use some http framework, like go-restful

**Comment by nowinkeyy** [comment]
> I don’t understand what this PR is used for. You need to add a more detailed explanation.

The database and object-storage deploy the same monitoring service respectively, and the service code needs to be split, and the code optimization has not progressed.

File: service/database/deploy/manifests/deploy.yaml [hunk 2]
```diff
@@ -37,17 +37,19 @@ spec:
...
         image: ghcr.io/labring/sealos-database-service:latest
         imagePullPolicy: Always
         name: database-monitor
         ports:
         - containerPort: 9090
           protocol: TCP
         resources:
+        limits:
+        cpu: 500m
+        memory: 1024Mi
```
Comment (lingdie) [inline service/database/deploy/manifests/deploy.yaml:49]: fix this.

**Review by lingdie — CHANGES_REQUESTED** [review]

**Review by lingdie — APPROVED** [review]

**Pull request merged**